### ISSUE TITLE
v4.4.0-beta1 Attempted to load class "ProblemNormalizer" from namespace not found

### ISSUE THREAD

**Issue by PhilETaylor** [symfony/symfony#34342]
**Symfony version(s) affected**:  v4.4.0-beta1

**Description**  
Took my live production app that works in 4.3 and upgraded to 4.4.0-BETA1

On first look - everything worked fine - shock! 

Then I tried to call a service (AuditService) which is injected using DI.

The issues looks to be deep in Symfony, might beyond my pay grade sorry. 

**How to reproduce**  
This is the worse issue report ever from me - no idea how to replicate outside my project - yet.. 

Im happy (and responsive) to questioning....

**Possible Solution**  
No idea... I suck. Im still debugging... 

**Additional context**  
![Screenshot 2019-11-12 at 18 46 30](https://user-images.githubusercontent.com/400092/68700438-d2445a80-057c-11ea-888c-b3c657c4beeb.png)

**Comment by PhilETaylor** [comment]
related to https://github.com/symfony/symfony/pull/34288/files

**Comment by nicolas-grekas** [comment]
`ProblemNormalizer` is a class of symfony/serializer 4.4
Can you check that's the version you have?

**Comment by PhilETaylor** [comment]
in composer.json was `  "symfony/serializer-pack": "*",` which resolved to 
```
 symfony/serializer: v4.3.7
 symfony/serializer-pack: v1.0.2
```

:-(

**Comment by PhilETaylor** [comment]
If I hammer it with 

```
"symfony/serializer-pack": "*",
"symfony/serializer": "4.4.*",
```
in composer.json then I get installed:

```
    symfony/serializer: v4.4.0-BETA1
    symfony/serializer-pack: v1.0.2
```

which is good enough for me tonight, but I still think `"symfony/serializer-pack": "*",` should have resolved to the correct 4.4.0-BETA1 version of ` symfony/serializer`

**Comment by yceruto** [comment]
As the Serializer component is optional, we could remove the service pointing to `ProblemNormalizer` if it doesn't exist, wdyt?

**Comment by nicolas-grekas** [comment]
@yceruto that's the job of the compiler.
@PhilETaylor we'll add a conflict rule with <4.4.

**Comment by PhilETaylor** [comment]
@nicolas-grekas No worries - sorry for a rambling unhelpful unprofessional issue post, just wanted it flagged quickly after the release in case it was something bigger. 

Showing my confidence in your b/c promise, the live service at https://manage.mysites.guru/ is now running on 4.4.0-BETA1 with 1000s of users :)

**Comment by yceruto** [comment]
Fixed in https://github.com/symfony/symfony/pull/34343

**Comment by PhilETaylor** [comment]
> Fixed in #34343

🎉🎉🎉 You guys rock
